migrations/20170905160630_add_has_caption_config_option_in_form_attributes.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddHasCaptionConfigOptionInFormAttributes extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+     *
+     * The following commands can be used in this method and Phinx will
+     * automatically reverse them when rolling back:
+     *
+     *    createTable
+     *    renameTable
+     *    addColumn
+     *    renameColumn
+     *    addIndex
+     *    addForeignKey
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function up()
+    {
+		$config = json_encode(array("hasCaption"=> true));
```

Comment:
We usually prefer the `["hasCaption" => true]` format for arrays

Reply:
Yup. Sorry about that. Fixed